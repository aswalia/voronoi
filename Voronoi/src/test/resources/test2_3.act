Vor:
lft and rgt: (35.0,13.0) (35.0,32.0)
lft and rgt: (37.0,39.0) (37.0,47.0)
lft and rgt: (35.0,32.0) (42.0,3.0)
lft and rgt: (37.0,47.0) (42.0,3.0)
lft and rgt: (35.0,13.0) (37.0,11.0)
lft and rgt: (34.0,36.0) (37.0,39.0)
lft and rgt: (34.0,36.0) (35.0,32.0)
lft and rgt: (31.0,30.0) (34.0,40.0)
lft and rgt: (34.0,40.0) (37.0,47.0)
lft and rgt: (35.0,32.0) (37.0,11.0)
lft and rgt: (34.0,40.0) (37.0,39.0)
lft and rgt: (37.0,11.0) (42.0,3.0)
lft and rgt: (31.0,30.0) (34.0,36.0)
lft and rgt: (34.0,36.0) (34.0,40.0)
lft and rgt: (35.0,32.0) (37.0,39.0)
lft and rgt: (31.0,30.0) (35.0,13.0)
lft and rgt: (37.0,39.0) (42.0,3.0)
lft and rgt: (31.0,30.0) (35.0,32.0)
lft and rgt: (35.0,13.0) (42.0,3.0)
