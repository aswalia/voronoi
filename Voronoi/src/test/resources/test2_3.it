node
node
node-point:(31.0,30.0) (34.0,36.0)
node-point:(34.0,40.0) (35.0,13.0) 
node
node-point:(35.0,32.0) (37.0,11.0) 
lft-point:(37.0,39.0)
node-point:(37.0,47.0) (42.0,3.0)

