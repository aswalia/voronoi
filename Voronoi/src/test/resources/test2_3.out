Vor:
(32.0,45.0) (36.666666666666664,43.0)
lft and rgt: (34.0,40.0) (37.0,47.0)
(36.666666666666664,43.0) (35.0,38.0)
lft and rgt: (34.0,40.0) (37.0,39.0)
(22.5,38.0) (35.0,38.0)
lft and rgt: (34.0,36.0) (34.0,40.0)
(35.0,38.0) (38.1,34.9)
lft and rgt: (34.0,36.0) (37.0,39.0)
(38.1,34.9) (31.833333333333332,33.333333333333336)
lft and rgt: (34.0,36.0) (35.0,32.0)
(22.5,38.0) (31.833333333333332,33.333333333333336)
lft and rgt: (31.0,30.0) (34.0,36.0)
(31.833333333333332,33.333333333333336) (37.25,22.5)
lft and rgt: (31.0,30.0) (35.0,32.0)
(37.25,22.5) (24.5,19.5)
lft and rgt: (31.0,30.0) (35.0,13.0)
(37.25,22.5) (46.5,22.5)
lft and rgt: (35.0,13.0) (35.0,32.0)
(46.5,22.5) (16.833333333333332,-7.166666666666668)
lft and rgt: (35.0,13.0) (37.0,11.0)
(16.833333333333336,-7.166666666666666) (33.5,4.5)
lft and rgt: (35.0,13.0) (42.0,3.0)
(67.5,24.5) (16.833333333333332,-7.166666666666668)
lft and rgt: (37.0,11.0) (42.0,3.0)
(71.29439252336448,25.41588785046729) (67.5,24.5)
lft and rgt: (35.0,32.0) (42.0,3.0)
(67.5,24.5) (46.5,22.5)
lft and rgt: (35.0,32.0) (37.0,11.0)
(197.9,43.0) (71.29439252336448,25.41588785046729)
lft and rgt: (37.0,39.0) (42.0,3.0)
(38.1,34.9) (71.29439252336448,25.41588785046729)
lft and rgt: (35.0,32.0) (37.0,39.0)
(61.5,27.5) (197.9,43.0)
lft and rgt: (37.0,47.0) (42.0,3.0)
(36.666666666666664,43.0) (197.9,43.0)
lft and rgt: (37.0,39.0) (37.0,47.0)
(27.5,36.5) (22.5,38.0)
lft and rgt: (31.0,30.0) (34.0,40.0)
